tensorflow_datasets/translate/flores.py

```diff
@@ -174,6 +174,9 @@ def _generate_examples(self, source_file, target_file):
             len(source_sentences), len(target_sentences), source_file,
             target_file)
 
+    assert len(target_sentences) == 0, "Dataset file downloaded is empty"
+    assert len(source_sentences) == 0, "Dataset file downloaded is empty"
```

Comment:
Have you try to generate the dataset with this change ?
If the dataset is generated without error, it means the dataset do not contains any examples, which is a bug.

Reply:
@Conchylicultor : the dataset does get's generated but if I remove the assert. sorry for the issue.
the gists are https://gist.github.com/dhirensr/6de14800140e1780de3eaaa20a693c76, https://gist.github.com/dhirensr/46eca5ef6b389fee3e635738b1276200 
also the number of samples is the same so I should just update the patch right?

Reply:
Are you sure you regenerated the dataset correctly ? The gist are both corresponding to v1.0.0, so it seems you haven't generated it with your changes

Reply:
@Conchylicultor : it's the same ! I just generated the files without changing the version number,but when I did it afterwards also the data size is coming the same. so I am sure it's the same.
should I just remove the tests and update the version to 1.0.1?

Reply:
I am not sure that you generated the dataset from the Github TFDS clone, because the dataset_info.json you copied do not contains some of the most recent additions (ex: dataset size).
Could you try to generate the data with your updates from the TFDS cloned version ?

Reply:
@Conchylicultor : I updated the gist and I am using the TFDS Cloned version and on branch flores_checksum_update to generate the dataset_info.json.
As you can see in the gist only the version gets updated
also on the terminal print I can see one extra thing which is not there in dataset_info.json and that's the download_size. are you referring to that?

Reply:
Thank you for the update. This looks good. I'll merge this soon.

Reply:
Actually, I still do not understand how the generation can succeed if you haven't removed the
`assert len(target_sentences) == 0`.
Could you add a `print(len(target_sentences))` next to your assert and copy the full generation logs in a gist. This looks suspicious to me.

The only way generation succeed is if the number of examples is empty, which doesn't seems to be the case by looking at the gist.
Additionally your gist show `1.0.1`, but the version here is `1.1.0` ?

Reply:
@Conchylicultor : I have pushed the code now in this branch! was waiting for you to review the gist files